zhazam/Modules/Onboarding/Model/OnboardingPage.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  OnboardingPage.swift
+//  zhazam
+//
+//  Created by Nurbek Ismagulov on 4/19/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import Foundation
+
+enum OnboardingType {
+    case interfaceStyle
+    case finish
+    
+    static let all: [OnboardingType] = [.interfaceStyle, .finish]
```

Comment:
You can use CaseIterable protocol that automatically generates an array property of all cases in an enum